build_annoy_forest.py

```diff
@@ -0,0 +1,20 @@
+from navec import Navec
+from annoy import AnnoyIndex
+
+# load navec vector file and extract vocabulary
+path = 'parser_tool/static/js/src/data/navec_hudlit_v1_12B_500K_300d_100q.tar'
+navec = Navec.load(path)
+vocabulary = navec.vocab.words
+
+# build annoy forest 
+dim = 300
+tree_count = 100
+forest = AnnoyIndex(dim, 'angular') 
+
+for i, word in enumerate(vocabulary):
+    forest.add_item(i, navec[word])
+
+forest.build(tree_count) 
+
+# save forest
+forest.save('parser_tool/static/js/src/ANNOY_tree.ann')
```

Comment:
Same thing here -- let's store the index in `parser_tool/data` since it's not intended to be public (e.g. directly downloadable, which it would be if it is' in the static folder).